README.md

```
@@ -1,3 +1,5 @@
+[![MseeP.ai Security Assessment Badge](https://mseep.net/pr/hexsleeves-tailscale-mcp-badge.png)](https://mseep.ai/app/hexsleeves-tailscale-mcp)
```

markdownlint-cli2 (0.22.1)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)